r/R/dplyr-summarize.R

```diff
@@ -161,6 +166,18 @@ summarize_eval <- function(name, quosure, ctx, recurse = FALSE) {
     return()
   }
 
+  if ("median" %in% funs_in_expr) {
+    expr <- wrap_median(expr, hash)
+    # TODO: Remove this warning after median() returns an exact median
+    # (ARROW-14021)
+    warn(
```

Comment:
I wonder if it would make more sense to move this warning to `agg_funcs$median` since we handle most other "arrow not supported" options in those functions.

Reply:
I wasn't sure that would work, but I guess it does. Done.